CHANGELOG.md

```diff
@@ -0,0 +1,669 @@
+- Accept &QuerierWrapper not &Querier in helpers [\#390](https://github.com/CosmWasm/cw-plus/issues/390)
+- Use builder pattern for App init [\#388](https://github.com/CosmWasm/cw-plus/issues/388)
+- Idea: item query helper storage helper [\#376](https://github.com/CosmWasm/cw-plus/issues/376)
+- Why you use `Addr` as keys in Maps? [\#295](https://github.com/CosmWasm/cw-plus/issues/295)
+- Add SnapshotItem to storage-plus [\#193](https://github.com/CosmWasm/cw-plus/issues/193)
+- Fix lifetime of MultiIndex/UniqueIndex to be able to accept &str [\#232](https://github.com/CosmWasm/cw-plus/issues/232)
+- Unify multisig structs Member and VoterResponse [\#151](https://github.com/CosmWasm/cw-plus/issues/151)
+
+**Merged pull requests:**
+
+- admin and hooks return Response\<C\> in execute\_\* [\#417](https://github.com/CosmWasm/cw-plus/pull/417) ([ethanfrey](https://github.com/ethanfrey))
+- Release 0.9.0 [\#416](https://github.com/CosmWasm/cw-plus/pull/416) ([ethanfrey](https://github.com/ethanfrey))
+- Add send and sendFrom to cw20-base helpers.ts [\#415](https://github.com/CosmWasm/cw-plus/pull/415) ([orkunkl](https://github.com/orkunkl))
+- Add doc entry about key usage in maps [\#413](https://github.com/CosmWasm/cw-plus/pull/413) ([maurolacy](https://github.com/maurolacy))
+- Add query helpers to Item and Map and use them in cw4 helpers [\#412](https://github.com/CosmWasm/cw-plus/pull/412) ([ethanfrey](https://github.com/ethanfrey))
+- Update Cargo.toml files to reference new repo name [\#411](https://github.com/CosmWasm/cw-plus/pull/411) ([ueco-jb](https://github.com/ueco-jb))
+- Snapshot item [\#409](https://github.com/CosmWasm/cw-plus/pull/409) ([maurolacy](https://github.com/maurolacy))
+- cw20-base: upgrade helper.ts to cosmjs 0.26.0 [\#406](https://github.com/CosmWasm/cw-plus/pull/406) ([spacepotahto](https://github.com/spacepotahto))
+- CW1-whitelist execute multitest [\#402](https://github.com/CosmWasm/cw-plus/pull/402) ([ueco-jb](https://github.com/ueco-jb))
+- Implementing all messages handling in mutlitest App [\#398](https://github.com/CosmWasm/cw-plus/pull/398) ([hashedone](https://github.com/hashedone))
+- Make it easier to assert events on reply statements [\#395](https://github.com/CosmWasm/cw-plus/pull/395) ([ethanfrey](https://github.com/ethanfrey))
+- Add helpers to check events [\#392](https://github.com/CosmWasm/cw-plus/pull/392) ([ethanfrey](https://github.com/ethanfrey))
+- Switching from String to anyhow::Error for error type in multi-test [\#389](https://github.com/CosmWasm/cw-plus/pull/389) ([hashedone](https://github.com/hashedone))
+
+## [v0.8.1](https://github.com/CosmWasm/cw-plus/tree/v0.8.1) (2021-08-26)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.8.0...v0.8.1)
+
+**Implemented enhancements:**
+
+- Consider replacing `String` errors with `anyhow::Error` in interfaces [\#361](https://github.com/CosmWasm/cw-plus/issues/361)
```

Comment:
How does it differentiate between "Implemented enhancements" and "Closed issues"?
Are there other tags we can add to issues/PRs to make them more meaningful in the changelog?

Reply:
I'm pretty sure `enhancements` are read from issue tags, like here: https://github.com/CosmWasm/cw-plus/issues/325

Reply:
Cool. We should add those labels to this repo and start using them

Reply:
I've added couple more labels.